Sources/DotNetGraph/Compiler/DotCompiler.cs

```diff
@@ -224,7 +222,9 @@ private void CompileAttributes(StringBuilder builder, ReadOnlyCollection<IDotAtt
 
         private static string FormatString(string value)
         {
-            var result = value.Replace("\"", "\\\"");
+            var result = value
+                .Replace("\\", "\\\\")
+                .Replace("\"", "\\\"");
```

Comment:
To answer your [question](https://github.com/vfrz/DotNetGraph/issues/13#issuecomment-619546581), I was thinking about this: `.Replace(Environment.NewLine, @"\n")`.

Notice, however, that `\n` is not a C# line break char - it is a string literal consisting of two characters: a backslash and n.

Reply:
Ok so the goal here is to remove any platform/environment-specific line break to `\n` so it's consistent?

Reply:
Let me clarify. In C# '\n' is a single character (LF), right? So this is not about replacing CRLF ("\r\n") with LF. It is about replacing CRLF with two following characters: @"\n" (a backslash and an n letter).

So this text:

```
First line
Second line
```

Should appear in the DOT label as:

`Firstline\nSecondline`

Reply:
You should see what I mean when you paste the following graph [here](http://www.webgraphviz.com/):

```
graph G {
    my_node [label="First line\nSecondLine"];
}
```

Reply:
Ok, I got it now, I'll make the changes later that day.

Reply:
I've made the changes, I've replace both \r\n and \n to `\\n` so it should works on linux and windows.

Reply:
Perfect, thanks! :)